pkg/skaffold/build/gcb/kaniko.go

```diff
@@ -36,6 +36,7 @@ func (b *Builder) kanikoBuildSpec(artifact *latest.KanikoArtifact, tag string) (
 	kanikoArgs := []string{
 		"--destination", tag,
 		"--dockerfile", artifact.DockerfilePath,
+		"--target", artifact.Target,
```

Comment:
Although it works well with an empty target, I'd rather add the `--target` only if the target is set. That feels better. You'd have to test both cases then.

Reply:
thanks for your advice.